src/components/existing-request/existing-request-display.vue

```diff
@@ -194,16 +194,10 @@
             :nrNum="nr && nr.nrNum"
             :approvedName="approvedName && approvedName.name"
             :emailAddress="nr && nr.applicants && nr.applicants.emailAddress"
-            :showIncorporateNowButton="showIncorporateNowButton"
-            :showRegisterButton="showRegisterButton"
-            :showAlterNowButton="showAlterNowButton"
-            :isAllowAlterOnline="isAlterOnline(nr.requestTypeCd)"
-            :showOpenExternalIcon="showOpenExternalIcon"
-            :showGoToSocietiesButton="showGoToSocietiesButton"
```

Comment:
I moved these (and the supporting getters/methods below) to the sub-component itself.